build-aux/.functions/log_group

```diff
@@ -0,0 +1,16 @@
+autoload -Uz log_info
+
+if (( ! ${+_log_group} )) typeset -g _log_group=0
+
+if (( ${+CI} )) {
+  if (( _log_group )) {
+    print "::endgroup::"
+    typeset -g _log_group=0
+  }
+  if (( # )) {
```

Comment:
Is this a zsh-ism?

Reply:
What specifically?

Reply:
Yeah that's ZSH - `${#}` gives you the number of positional arguments to a function and arithmetic checks in ZSH do not need the variable notation (`if (( VAR ))` is equal in function to `if (( ${VAR} ))`).